### PR TITLE
Wishlist page merge ready

### PR DESCRIPTION
**Pull request by b0llu**
Only check wishlist.html and wishlist.css

image
image

### CODE REVIEW

**Review by ParthParolekar — APPROVED** [review]
Looks good to me!

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Looks Good

File: cart-page/cart.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                <div class="footer-icon-container">
+                    <a href="https://github.com/B0llu"><i class="fab fa-github"></i></a>
+                    <a href="https://twitter.com/TheBestDhruv"><i class="fab fa-twitter"></i></a>
+                </div>
+            </div>
+        </footer>
+    </section>
+    <script src="../ecom-js/theme.js"></script>
+</body>
+
```
Comment (iRohitGaur) [inline cart-page/cart.html:169]: Suggestion: Put comments to segregate the sections. Eg: "SECTION XYZ START"

Comment (b0llu) [reply cart-page/cart.html:169]: Yeah I will do that.

File: ecom-css/cart-grid-layout.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: auto repeat(3, 1fr) auto;
+  grid-template-areas:
+    "nav nav nav nav nav nav"
+    "contentarea contentarea contentarea contentarea contentarea contentarea"
+    "contentarea contentarea contentarea contentarea contentarea contentarea"
+    "contentarea contentarea contentarea contentarea contentarea contentarea"
+    "footerarea footerarea footerarea footerarea footerarea footerarea";
+}
```
Comment (iRohitGaur) [inline ecom-css/cart-grid-layout.css:16]: no need to repeat them if they are same. you can do this instead: 

"nav"
"contentarea"
"footerarea"

File: ecom-css/media-query.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+  .landing-container {
+    grid-template-columns: repeat(1, 25rem) repeat(2, 1fr);
+    grid-template-areas:
+      "nav nav nav"
+      "sidebar contentarea contentarea"
+      "sidebar contentarea contentarea"
+      "sidebar contentarea contentarea"
+      "footerarea footerarea footerarea";
+  }
```
Comment (iRohitGaur) [inline ecom-css/media-query.css:14]: same thing can be applied here

File: ecom-css/media-query.css [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+@media screen and (max-width: 450px) {
+  .landing-container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-areas:
+      "nav nav nav"
+      "contentarea contentarea contentarea"
+      "contentarea contentarea contentarea"
+      "contentarea contentarea contentarea"
+      "footerarea footerarea footerarea";
```
Comment (iRohitGaur) [inline ecom-css/media-query.css:68]: and here..

File: ecom-css/product-grid-layout.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+.landing-container {
+  min-height: 100%;
+  display: grid;
+  grid-template-columns: repeat(2, 20rem) repeat(4, 1fr);
+  grid-template-areas:
+    "nav nav nav nav nav nav"
+    "sidebar sidebar contentarea contentarea contentarea contentarea"
+    "sidebar sidebar contentarea contentarea contentarea contentarea"
+    "sidebar sidebar contentarea contentarea contentarea contentarea"
+    "footerarea footerarea footerarea footerarea footerarea footerarea";
```
Comment (iRohitGaur) [inline ecom-css/product-grid-layout.css:14]: and here..

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge with updates 👍🏻

**Pull request merged**